src/types/onyx/Transaction.ts

```diff
@@ -25,6 +37,7 @@ type Transaction = {
         source?: string;
         state?: ValueOf<typeof CONST.IOU.RECEIPT_STATE>;
     };
+    routes: Routes;
```

Comment:
Not all transactions have routes
```suggestion
    routes?: Routes;
```

Reply:
good catch!